docs/src/main/sphinx/sql/alter-table.rst

```diff
@@ -29,6 +29,22 @@ The optional ``IF EXISTS`` (when used before the column name) clause causes the
 
 The optional ``IF NOT EXISTS`` clause causes the error to be suppressed if the column already exists.
 
+.. _alter-table-set-properties:
+
+SET PROPERTIES
+^^^^^^^^^^^^^^
+
+The ``ALTER TABLE SET PROPERTIES``  statement followed by some number
+of ``property_name`` and ``expression`` pairs applies the specified properties
+and values to a table. Ommitting an already-set property from this
+statement leaves that property unchanged in the table.
+
+A property in a ``SET PROPERTIES`` statement can be set to ``DEFAULT``, which
+reverts its value back to the default in that table.
+
+Support for ``ALTER TABLE SET PROPERTIES`` varies between
+connectors. Refer to the connector documentation for more details.
```

Comment:
Should the term "connector documentation" link to https://trino.io/docs/current/connector.html

Reply:
Good idea, added